coderedcms/models/wagtailsettings_models.py

```diff
@@ -273,7 +279,12 @@ class GeneralSettings(BaseSetting):
         blank=True,
         max_length=255,
         verbose_name=_('From email address'),
-        help_text=_('The default email address this site appears to send from. For example: "sender@example.com" or "Sender Name <sender@example.com>" (without quotes)'),
+        help_text=_(
+            '''
+            The default email address this site appears to send from.
+            For example: "sender@example.com" or "Sender Name <sender@example.com>" (without quotes)
+            '''
```

Comment:
Triple quotes will preserve spaces and newlines, use normal quotes here instead.